chat_shell/chat_shell/main.py

```
@@ -0,0 +1,79 @@
+
+
+# Create application instance
+app = create_app()
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run(
+        "app.main:app",
+        host="0.0.0.0",
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)